src/Language/PureScript/Publish/ErrorsWarnings.hs

```diff
@@ -148,9 +148,8 @@ displayUserError e = case e of
             , "version."
             ])
         , spacer
-        , para "Note: tagged versions must be in one of the following forms:"
+        , para "Note: tagged versions must be in the following form:"
```

Comment:
Would it make sense to remove the bullet point now that there's only one form listed?